combat-trainer.lic

```diff
@@ -2487,6 +2492,14 @@ class TrainerProcess
       bput('meditate research earthquake', 'You clear ')
     when /^Barb Research Utility$/i
       bput('meditate research prediction', 'You clear ')
+    when /^Berserk Landslide$/i
+      if DRStats.mana >= @barb_buffs_inner_fire_threshold
+        DRCA. activate_barb_buff?('Landslide')
```

Comment:
Does this work with a space in `DRCA. act.......`?

Reply:
I wondered about that space, but it's working fine. Is that space not supposed to be there? I can take it out.